cmd/crc/cmd/start.go

```diff
@@ -321,3 +329,26 @@ func checkDaemonStarted() error {
 	}
 	return daemonclient.CheckIfOlderVersion(v)
 }
+
+func portFallbackWarning() string {
+	var fallbackPortWarning string
+	httpScheme := "http"
+	httpsScheme := "https"
+	fallbackPortWarningTmpl := `
+Warning: Port %d is used for OpenShift %s routes instead of the default
+You have to add this port to %s URLs when accessing OpenShift application,
+such as %s://myapp.apps-crc.testing:%d
```

Comment:
Warning that the console will be non-functional would be useful here as well. Especially as one message just before the warning is 
```
The server is accessible via web console at:
  {{ .ClusterConfig.WebConsoleURL }}
```
A follow-up would be to remove this message when the https is overridden.

Reply:
Address as a follow-up